src/main/java/mate/academy/spring/service/mapper/CinemaHallMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.dto.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.CinemaHallResponseDto;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
```suggestion
@Component
```

task: Implement mappers for each entity. Mark your mappers as @Component